blueprint/api.apib

```diff
@@ -2209,6 +2268,59 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Stripe Auth Resource)
 + include JSON API Version
 
+## StripeCard (object)
++ data(StripeCard Resource)
++ include JSON API Version
```

Comment:
This is still why it's failing. Trying to hit a `StripeCard Resource` that doesn't exist.